go/vt/tabletserver/querylogz_test.go

```diff
@@ -44,11 +45,18 @@ func TestQuerylogzHandler(t *testing.T) {
 	logStats.CacheMisses = 2
 	logStats.CacheInvalidations = 3
 	logStats.TransactionID = 131
+	logStats.ctx = callerid.NewContext(
+		context.TODO(),
```

Comment:
This should be context.Background(). Apparently TODO is to be used only for non-text code.